src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
 </figure>
 
-Select a Layout Shift to view its details in the **Summary** tab. To visualize where the shift itself occurred, hover over the **Moved from** and **Moved to** fields.
+While the [Core Web Vitals report in Search Console](https://support.google.com/webmasters/answer/9205520) shows the big picture of your website's performance, you can still drill down into specific pages that need attention. With Search Console, you can also:
 
-**Debug interaction readiness with Total Blocking Time in the footer**
+- Identify individual pages that need improvement and those that currently provide a good user experience.
+- Get granular data on performance by URL grouped by status, metric, and groups of similar web pages (such all product detail pages on an e-commerce website, for example).
+- Get detailed reports that bucket URLs in each user experience quality category for both mobile and desktop.
 
-The Total Blocking Time (TBT) metric can be measured in lab tools and is an excellent proxy for First Input Delay. TBT measures the total amount of time between [First Contentful Paint (FCP)](/fcp/) and [Time to Interactive (TTI)](/tti/) where the main thread was blocked for long enough to prevent input responsiveness. Performance optimizations that improve TBT in the lab should improve FID in the field.
+{% Aside 'caution' %}
+The data in Search Console is different from what's shown in CrUX dashboard. This is because Search Console organizes information by URL, whereas CrUX Dashboard organizes data by origin. If you have few poorly performing URLs, but those URLs receive a large portion of overall traffic, Search Console will only show a few poorly performing URLs while CrUX dashboard will show a high percentage of poor user experiences.
+{% endAside %}
+
+#### Analyze performance by page with PageSpeed Insights
 
 <figure class="w-figure">
-  {% Img src="image/admin/WufuLpvrZfgbRn70C74V.png", alt="Total Blocking Time displayed in the footer of the DevTools performance panel", width="800", height="517", class="w-screenshot" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/qobynTXHVaEPD2Ng8ZLb.png", alt="A screenshot of how PageSpeed Insights portrays CrUX data for a URL's Core Web Vitals. Each of the Core Web Vitals is displayed separately, while grouping each Core Web Vital in the 'Good', 'Needs Improvement', and 'Poor' thresholds for the last 28 days.", width="800", height="268" %}
 </figure>
 
-TBT is now shown in the footer of the Chrome DevTools **Performance** panel when you measure page performance:
+PageSpeed Insights displays a subset of CrUX data at the URL level covering the last 28 days of user experience data at the 75th percentile. This means that if 75% of user experiences meet the [threshold set for a given metric](/defining-core-web-vitals-thresholds/), then the experience is considered "good".
```

Comment:
Everything you say in this sentence is true, but it's also true of all tools that display CrUX data (i.e. all CrUX data is p75 over a 28-day window).

By calling it out specifically here I'm a bit concerned that it might make it seem like the other tools _don't_ show p75 data over a 28-day period.